pkg/model/message.go

```diff
@@ -9,4 +9,5 @@ type Message struct {
 type Messages struct {
 	Language string
 	Messages []Message
```

Comment:
Messages.Messages ?

Reply:
I used this struct https://github.com/expect-digital/translate/issues/4

Reply:
There is some stuttering, but it's ok. I tried different names, but none improves clarity or readability. I was inspired by https://pkg.go.dev/golang.org/x/text@v0.5.0/message/pipeline#Messages